content/en/blog/2022/OpenTelemetry-In-Practice-August.md

```diff
@@ -0,0 +1,55 @@
+---
+title: OpenTelemetry in Practice - Kubernetes & the Collector
+linkTitle: OTel in Practice
+date: 2022-08-11
+author: Rynn Mancuso
```

Comment:
feel free to add a link to your github account

Reply:
@svrnm How would one add a link to their GitHub account within this frontmatter template? Link or resource is appreciated!

Reply:
Use markdown, like this:

```yaml
author: '[Ruben Vargas](https://github.com/rubenvp8510)'
```

which will show up like this: https://opentelemetry.io/blog/2022/k8s-metadata/